### DISCUSSION TITLE
Issue template forms docs

### DISCUSSION THREAD

**Discussion by nt4f04uNd** [community/community#4120] [Other Feature Feedback, Questions, & Ideas] [answered]
I cannot find any announcements/documentation on new issue template form format
https://github.com/microsoft/terminal/issues/new?assignees=&labels=&template=Bug_Report.yml

**Comment by michellemerrill** [comment, marked as answer]
Hi @nt4f04uNd Issue forms are currently in private alpha. If you've got an open source project that you maintain that you think they'd be really useful for, feel free to shoot Luke Hefson [an email](mailto:lukehefson@github.com) and he can get you onboarded into the private alpha!
